### DISCUSSION TITLE
Any guidance on getting mookme to work under Windows?

### DISCUSSION THREAD

**Discussion by efenderbosch-atg** [Escape-Technologies/mookme#124] [Q&A]
Team members using Ubuntu or MacOS are fine, but our hooks are not running under Windows.

**Comment by emercadoAtg** [comment]
use .git/config to run a pre commit hook

**Reply by emercadoAtg** [reply]
this will not run  mookme but either way we can run the hooks 
 something like : 
 [hooks]
pre-stage = .git/hooks/pre-commit
